### DISCUSSION TITLE
Incorrect timing

### DISCUSSION THREAD

**Discussion by bullhorn-admin** [openai/whisper#2060] [General]
Hello, in the following example I say "Purple submarine"  between 7.4s-8.6s
<img width="1243" alt="Screenshot 2024-03-03 at 10 05 56 PM" src="https://github.com/openai/whisper/assets/130067123/a2e9ab57-5048-4bf5-8038-22cb4d3e4058">
[0bcad4b4-04e3-4794-907e-1aa59b3ff9b2-2.mp3.zip](https://github.com/openai/whisper/files/14476039/0bcad4b4-04e3-4794-907e-1aa59b3ff9b2-2.mp3.zip)

However openai whisper thinks that I stopped at 7.0

```
{
    "text": "''Purple Submarine'",
    "task": "transcribe",
    "language": "english",
    "duration": 20.0,
    "segments": [
        {
            "id": 0,
            "seek": 0,
            "start": 0.0,
            "end": 7.0,
            "text": " ''Purple Submarine'",
            "tokens": [
                50364,
                12119,
                47,
                374,
                781,
                8511,
                6209,
                533,
                6,
                50714
            ],
            "temperature": 0.0,
            "avg_logprob": -0.8815329074859619,
            "compression_ratio": 0.7037037014961243,
            "no_speech_prob": 0.14761993288993835
        }
    ]
}

```
This is a big deal for me as I am unable to attribute it to the correct speaker.

Any ideas what it is all about?

Thank you,
Eugene

**Comment by ryanheise** [comment]
Settings the `word_timestamps` option to `True` tends to give more accurate timestamps. e.g. From the command line, add the option `--word_timestamps True`

**Comment by misutoneko** [comment]
Always be using --suppress_tokens

Well, that's what I do anyway.  Small model with --suppress_tokens 50364 seems to get it right in this particular case.

**Comment by bullhorn-admin** [comment]
I am using OpenAI version in their cloud. I assume there is no way to try what you suggested there, so I need to switch to self hosted version?
